modules/contact-form/grunion-form-view.php

```diff
@@ -14,9 +17,12 @@
  */
 $max_new_fields = apply_filters( 'grunion_max_new_fields', 5 );
 
+if ( ! $asset_tools instanceof Asset_Tools ) {
+	$asset_tools = new Asset_Tools();
```

Comment:
this looks weird. I wonder if we should share via the global scope `$jetpack_get_file_url_for_environment` or `$jetpack_asset_tools`